tox.ini

```diff
@@ -18,8 +18,12 @@ python =
 description = run api tests
 deps =
     fastapi
+    tortoise-orm
+    pypika-tortoise
```

Comment:
I don't think we need pypika-tortoise and asyncpg for tests, but let me know.

Reply:
Oh asyncpg is needed for pgsql connection (docs metioned it .. will try without it) .. and pypika .. my initial plan was to use pg's inbuilt sha256 encoder for encoding the password before storing it into db... for that i added pypika .. will be removed